antora.yml

```diff
@@ -64,6 +64,7 @@ asciidoc:
     prod-id: eclipse-che
     prod-namespace: che
     prod-operator: che-operator
+    prod-operator-image-name: che-operator
```

Comment:
Why is this separate from `prod-operator`?

And if it needs to be separate, let's use `prod-operator` as the value for this until such time as it needs changing.

Reply:
because prod-operator gives me an incorrect value in downstream. It is the same case as with the prod and prod2 attribute

Reply:
Note that in CRW's CSV:

```
                      - name: OPERATOR_NAME
                        value: codeready-operator
                    command:
                      - /usr/local/bin/che-operator
```

So you'll want to have two variables in play -- one for the filename (if applicable in the docs) and another to refer to the operator as it's used in k8s/OCP.

Reply:
I talked about this with Nick. I believe its a way to go since I  just talking about the image name, not the object or cli names. The attribute for CRW will be added with my dedicated MR here:
https://gitlab.cee.redhat.com/red-hat-developers-documentation/red-hat-devtools/-/merge_requests/752